airgun/views/discoveryrule.py

```diff
@@ -59,6 +63,23 @@ class primary(SatTab):
         hosts_limit = TextInput(id='discovery_rule_max_count')
         priority = TextInput(id='discovery_rule_priority')
         enabled = Checkbox(id='discovery_rule_enabled')
+        AUTOCOMPLETE_LIST = "//ul[contains(@class,'autocomplete')]"
+
+        def fill(self, values):
+            was_change = False
+            if values:
+                search_value = values.pop('search')
```

Comment:
does `ENSURE_PAGE_SAFE` catch that situation as it do for generic search on other pages? If yes what widget does not trigger ensure_page_safe in current hierarchy?

Reply:
seems ENSURE_PAGE_SAFE does not catch that situation, the FilteredDropdown widget is the next widget to be filled.